rules-java-archives/addon/src/test/java/org/jboss/windup/rules/apps/java/archives/identify/test/SortedFileArchiveIdentifierTest.java

```diff
@@ -0,0 +1,83 @@
+package org.jboss.windup.rules.apps.java.archives.identify.test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.forge.addon.dependencies.builder.CoordinateBuilder;
+import org.jboss.windup.rules.apps.java.archives.identify.SortedFileArchiveIdentifier;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+/**
+ *
+ * @author Ondrej Zizka, ozizka at redhat.com
+ */
+public class SortedFileArchiveIdentifierTest
+{
+
+    public SortedFileArchiveIdentifierTest()
+    {
+    }
+
+
+    @Test
+    public void testGetCoordinateFromSHA1() throws FileNotFoundException
+    {
+        // TODO: I suggest to rename archive-metadata to archive-mapping. It's not metadata.
+        final File file = new File("target/test-nexus-data/central.archive-metadata.txt");
```

Comment:
Hey, could you provide me some context? What is the point of this? From my understanding, you've copied the data-file into the test dir as part of the build (duplicating the version of the nexus data artifact in two poms) just to test the SortedFileArchiveIdentifier. Why not just use the sample file that you used in the test below? (which is a lot smaller but would achieve the same goal?)